ui/components/app/permission-page-container/permission-page-container-content/permission-page-container-content.component.js

```diff
@@ -102,7 +105,7 @@ export default class PermissionPageContainerContent extends PureComponent {
         >
           <PermissionsConnectPermissionList
             permissions={selectedPermissions}
-            subjectName={subjectMetadata.origin}
+            subjectName={displayOrigin}
```

Comment:
Why was this changed?

Reply:
My bad! Reverted https://github.com/MetaMask/metamask-extension/pull/30092/commits/7abb5f844a3728a4cd4892764acc02606e813364